src/wallet/wallet.cpp

```diff
@@ -48,6 +48,8 @@ bool fSendFreeTransactions = DEFAULT_SEND_FREE_TRANSACTIONS;
  */
 CFeeRate CWallet::minTxFee = CFeeRate(DEFAULT_TRANSACTION_MINFEE);
 
+const uint256 CMerkleTx::abandonHash(uint256S("0000000000000000000000000000000000000000000000000000000000000001"));
```

Comment:
nit: I probably would do it also that way,... but, maybe a cleaner way would be to use `CWalletTx::mapValue` (a flexible kv store) which also would not require a db-scheme update.